account-integrations/safe/src/SafeWebAuthnPlugin.sol

```diff
@@ -92,12 +95,13 @@ contract SafeWebAuthnPlugin is BaseAccount {
             i += dataLen; // advance index
 
             // decode fixed length params (values to memory)
-            dataLen = 4 * 32; //lenFixedParams - 32; // -32 already read length
+            dataLen = paramLen - 32; // length already read
+            dataLen -= 2 * 2 * 32; // exclude fixed length arrays
```

Comment:
I'm not sure I'm understanding this change, can you explain what's going on here and how this leads to the same value as `dataLen = 4 * 32;`?

Reply:
The param length is the length of all fixed bytes including the length parameter (you can add this up from the fixed params or print out to see the value).
The first line subtracts the already-read length, the next line subtract two arrays of length 2 containing 32bytes.

Reply:
As discussed in call, clarified changes and updated one of the comments